docs/source/authn-guide/custom-ldap-replication.md

```diff
@@ -0,0 +1,82 @@
+# Authenticating via an External IDP a New User Created Using a Custom Authentication Script in an Opendj Replication Setup
```

Comment:
Add this as a header for the example . It should be ` Authenticating via an External IDP a New User Created Using a Custom Authentication Script in Kubernetes`